README.md

```diff
@@ -12,5 +12,20 @@
 
 ## Usage
 
+1. Put your config file in `conf/config.json`
```

Comment:
If I'm only using a single config, will this work? Looking at `synapse-balancer@.service` below, I see `DEPLOY_ENV` without a default value, so it would use the file `.json` as its config, right?

`start-synapse.sh` and `script/deploy` seem to have the same issue, where `$DEPLOY_ENV` is used without checking it has a value.